db/seeds.rb

```diff
@@ -1,4 +1,4 @@
 # Create a few default tags.
 Tag::DEFAULT_TAGS.each do |name|
-  Tag.create!(name: name)
+  Tag.find_or_create_by(name: name)
```

Comment:
We are running `rails db:seed` when building the docker image. In case there is existing sqlite3 and tags are already created in that DB we will get error of uniqueness for tags. To avoid uniqueness error  we have changed `create!` to `find_or_create_by`. We can also remove 'bang(!)' to avoid error. But I think `find_or_create_by` is more appropriate in this scenario.

Reply:
But wouldn't running db:seed require access to the DB during the build process?
I think another option would be to construct a secondary-image or script that takes over the seeding-process and whatever is needed before the first start of the application.